definitions.yaml

```diff
@@ -666,3 +666,84 @@ definitions:
       auth_token:
         type: string
         description: The newly created API token
+
+  # The list of all invitations.
+  V4InvitationsResponse:
+    type: array
+    description: List of members that belong to this organization
+    items:
+      $ref: '#/definitions/V4Invitation'
+
+  V4Invitation:
+    type: object
+    properties:
+      accepted:
```

Comment:
this is optional right? if it's pending for example

Reply:
Yes, it can be blank. But this is part of a response, so there is no requried or optional specification here.

Reply:
ok, was just not sure if there's a way to mention sth about optionality in the spec as otherwise an api user might expect there to be such a field coming back

Reply:
My intention here is that the field will come back, but it'll be empty if the invitation has not yet accepted.

Reply:
> Regarding returning accepted as an empty string when not accepted yet: I consider that a bad practice. I would leave it out (or second best: make it have null as a value) in that case.

I disagree here. Isn't it better to have a single type possible for that field? instead of it being a `null` or `string` ?

Reply:
This is very much a Go-minded argument. `null` as a special value for an attribute that would otherwise carry a string/a number/an array/an object is pretty common in JSON APIs and even Go clients should be able to deal with this. That said, only returning an attribute like `accepted` when it actually has a meaningful value would make this even simpler.